hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ListBucketResponse.java

```diff
@@ -39,6 +39,9 @@ public class ListBucketResponse {
   @XmlElement(name = "Bucket")
   private List<BucketMetadata> buckets = new ArrayList<>();
 
+  @XmlElement(name = "Owner")
+  private S3Owner owner;
```

Comment:
Unrelated, but S3Owner javadoc is not useful. We should fix it in another task.

    /**
     * TODO: javadoc.
     */
    @XmlAccessorType(XmlAccessType.FIELD)
    @XmlRootElement(name = "Owner")
    public class S3Owner {